redirect/example/main.tf

```diff
@@ -0,0 +1,15 @@
+provider "aws" {
+  region = "eu-west-1" # Ireland
+}
+
+module "redirect" {
+  source = "./.."
+
+  project     = "terraform-modules-redirect"
+  environment = "example"
+  target      = "https://www.google.com"
```

Comment:
seems like a perfect place to plug https://codequest.com/ :wink:

Reply:
👍